### PR TITLE
feat: added tabbar for mobile and cleaned folder structure

### LINKED ISSUE #5: Implement TabBar Navigation
TabBar is used to navigate on mobile devices.

TabBar - open

- [x] Add icons and labels
- [x] Opacity Animation when clicked
- [x] Closes when clicked on the same icon
- [x] Changes active Tab when clicking on another icon
- [x] Content components can be added inside tabs
- [x] SubNavigation inside "More" Tab

### PR DESCRIPTION
**Pull request by DerBeton**
Fixed #5

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
 
 function App() {
   return (
     <>
-  <SideBar />
-      
-     
+      <SideBar className="visible" />
+      <TabBar className="hidden" />
```
Comment (Cyril676) [inline src/App.jsx:12]: Remove className, not working inside components

File: src/components/SideBar/SideBar.jsx [hunk 2]
```diff
@@ -28,27 +28,27 @@ const SideBar = () => {
...
             <div onClick={ () => setOpen(false) } className={`w-full h-screen z-[100] ${open ? "pointer-events-auto" : "pointer-events-none"}`}></div>
             <aside className={`flex w-[900px] right-0 top-0 h-screen pointer-events-auto backdrop-blur backdrop-brightness-90 bg-darkblue/80 transition-all duration-500 ease-in-out ${open ? "translate-x-0" : "translate-x-[800px]"}`}>
                 <nav className="w-[100px] pt-10 pb-10 flex flex-col h-full bg-white/20 z-[999]">
-                <div className="flex flex-col items-center flex-grow-0 justify-start flex-1">
+                <div className="flex flex-col space-y-5 items-center flex-grow-0 justify-start flex-1">
                     <div className="">
                         <NavItem icon={<CableCarSVG className={NAV_ICON_CLASSES} />} title="Plants" togglePage={togglePage} page={page} open={open} />
                     </div>
-                    <div className="my-5">
+                    <div className="">
```
Comment (Cyril676) [inline src/components/SideBar/SideBar.jsx:39]: remove empty className

File: src/components/TabBar/TabBar.jsx
Comment (Cyril676) [inline src/components/TabBar/TabBar.jsx:1]: Could make use of sub-components (content, navigation) for better readability, possibly problems: having to pass data / props.

**Review by Cyril676 — COMMENTED** [review]
Seems to be OK expect e few small details

File: src/App.jsx (hunk 1 above)
Comment (DerBeton) [reply src/App.jsx]: Thanks removed with newest commit.

File: src/components/TabBar/TabBar.jsx
Comment (DerBeton) [reply src/components/TabBar/TabBar.jsx:1]: Good idea but probably to much work with passing props to parent and child components. I'll leave it for now.

**Pull request merged**